lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -3613,17 +3615,17 @@ void mme_app_update_paging_tai_list(
   switch (tai_list->typeoflist) {
     case TRACKING_AREA_IDENTITY_LIST_ONE_PLMN_NON_CONSECUTIVE_TACS:
       for (int idx = 0; idx < (num_of_tac + 1); idx++) {
-        p_tai_list->tai_list[idx].mcc_digit1 =
+        p_tai_list->tai_list[idx].plmn.mcc_digit1 =
```

Comment:
Since TAI is copied; can we use COPY_TAI or COPY_PLMN macros

Reply:
To be finished later because it requires changes in some other portions of the code as struct "tai_one_plmn_non_consecutive_tacs" is not supported by COPY_TAI

Reply:
Resolved